### PR TITLE
docs(examples): Add Tailwindcss

### PR DESCRIPTION
**Pull request by juwiragiye**
- docs(examples): Add Tailwindcss

### CODE REVIEW

File: examples/with-tailwindcss/README.md
Comment (kentcdodds) [inline examples/with-tailwindcss/README.md:3]:
```suggestion
Integrate Remix with tailwindcss.
```

File: examples/with-tailwindcss/README.md
Comment (kentcdodds) [inline examples/with-tailwindcss/README.md:9]:
```suggestion
```

File: examples/with-tailwindcss/README.md
Comment (kentcdodds) [inline examples/with-tailwindcss/README.md:11]: The "with-" prefix is unnecessary. Let's rename this folder.

```suggestion
[![Open in codesandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/remix-run/remix/tree/main/examples/tailwindcss)
```

File: examples/with-tailwindcss/README.md
Comment (kentcdodds) [inline examples/with-tailwindcss/README.md:15]:
```suggestion
This example shows how to use Tailwind CSS (v3.0) with Remix. It follows the steps outlined in the official [Remix Styling docs](https://remix.run/docs/en/v1/guides/styling#tailwind).
```

**Review by kentcdodds — CHANGES_REQUESTED** [review]
This looks great! Just a couple of changes

File: examples/tailwindcss/README.md
Comment (MichaelDeBoey) [inline examples/tailwindcss/README.md:9]:
```suggestion
[![Open in CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/remix-run/remix/tree/main/examples/tailwindcss)
```

File: examples/tailwindcss/README.md
Comment (MichaelDeBoey) [inline examples/tailwindcss/README.md:19]:
```suggestion
[Tailwind CSS](https:/tailwindcss.com)
```

File: examples/tailwindcss/styles/tailwind.css
Comment (MichaelDeBoey) [inline examples/tailwindcss/styles/tailwind.css:1]: This file isn't necessary as Tailwind will use this setup by default

**Review by MichaelDeBoey — CHANGES_REQUESTED** [review]

File: examples/tailwindcss/README.md
Comment (kentcdodds) [inline examples/tailwindcss/README.md:19]:
```suggestion
[Tailwind CSS](https://tailwindcss.com)
```

File: examples/tailwindcss/README.md
Comment (kentcdodds) [inline examples/tailwindcss/README.md:15]:
```suggestion
Relevant files:
- [package.json](./package.json) where the tailwind CLI is used.
- [tailwind.config.js](./tailwind.config.js) where tailwind is configured.
- [app/root.tsx](./app/root.tsx) where tailwind is imported.
```

File: examples/tailwindcss/styles/tailwind.css
Comment (kentcdodds) [reply examples/tailwindcss/styles/tailwind.css:1]: @juwiragiye could you delete this file?

**Review by kentcdodds — CHANGES_REQUESTED** [review]
Thanks for this! Just a few things need to be fixed then we can get this merged :)

**Comment by juwiragiye** [comment]
I deleted the file. I'm also doing a Prisma example.

**Comment by DevCeur** [comment]
I just tried out this example and I'm seen this in my console everytime I change some styles, is this expected or it's just my local environment 🤔

Screen Shot 2022-01-07 at 10 12 29 AM


BTW, in my `build/_assets` there's a lot of css files generated, I think this is what causes the above console.error

Screen Shot 2022-01-07 at 10 13 22 AM

**Comment by juwiragiye** [comment]
@DevCeur  I've been also having the same error every time I create a new project with npx create-remix@latest. It shows up during development but when you build for production, the error is gone. Also @ryansolid, the creator of SolidJS kept having the same when he was trying out Remix. Let's wait for @MichaelDeBoey  or @kentcdodds  to give feedback on this.

**Review by MichaelDeBoey — CHANGES_REQUESTED** [review]
@juwiragiye Please do the Prisma example in a different branch/PR

This way we can focus on separate examples in separate PRs

**Comment by juwiragiye** [comment]
@MichaelDeBoey  I did the Prisma example in a different branch/PR. I already pushed it.I'm waiting for approval.

File: examples/prisma/.gitignore
Comment (MichaelDeBoey) [inline examples/prisma/.gitignore:1]: @juwiragiye Can you please delete the Prisma example from this branch/PR?

**Review by MichaelDeBoey — CHANGES_REQUESTED** [review]

File: examples/prisma/.gitignore
Comment (juwiragiye) [reply examples/prisma/.gitignore]: @MichaelDeBoey I successfully deleted prisma example from this branch/PR

**Review by kentcdodds — APPROVED** [review]
Fantastic. Thanks!

**Pull request merged**
